src/components/base/accordion/Accordion.js

```diff
@@ -66,7 +67,7 @@ export function Item({
   const handleKeypress = (e) => {
     //it triggers by pressing the enter key
     if (e.key === "Enter" || e.keyCode === 13) {
-      onClick();
+      onClick(() => {});
```

Comment:
Hvorfor?

Reply:
Det er fordi der kommer en warning og det er ret irriterende at se på, så jeg ville bare fjerne den, uden at ændre for meget kode. Men jeg kan finde en anden løsning